pkg/network/protocols/postgres/statskeeper_test.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package postgres
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestStatKeeperProcessValidateSketches(t *testing.T) {
+	cfg := config.New()
+	cfg.MaxPostgresStatsBuffered = 100
+	s := NewStatkeeper(cfg)
+	for i := 0; i < 20; i++ {
+		s.Process(&EventWrapper{
+			EbpfEvent: &EbpfEvent{
+				Tx: EbpfTx{
+					Request_started:    1,
+					Response_last_seen: 10,
+				},
+			},
+			operationSet: true,
+			operation:    SelectOP,
+			tableNameSet: true,
+			tableName:    "dummy",
+		})
+	}
+
+	require.Equal(t, 1, len(s.stats))
+	for k, stat := range s.stats {
+		require.Equal(t, "dummy", k.TableName)
+		require.Equal(t, SelectOP, k.Operation)
```

Comment:
You are validating more than just the sketches in the test. If you wish to keep it that way, rename the test. If not, remove the redundant validations, such as the table name